splunk_add_on_ucc_framework/global_config_validator.py

```diff
@@ -224,10 +224,73 @@ def _validate_validators(self):
             entities = service["entity"]
             for entity in entities:
                 self._validate_entity_validators(entity)
+    @staticmethod
+    def _find_duplicates_in_list(_list) -> list:
+        dup = [x for x in _list if _list.count(x) > 1]
+        return dup if dup else False
```

Comment:
This method can be optimized to get the result about the duplicates in the list faster.

Reply:
Changed to use more efficient way